src/guide/introduction.md

```diff
@@ -316,7 +316,7 @@ In a large application, it is necessary to divide the whole app into components
 
 You may have noticed that Vue components are very similar to **Custom Elements**, which are part of the [Web Components Spec](https://www.w3.org/wiki/WebComponents/). That's because Vue's component syntax is loosely modeled after the spec. For example, Vue components implement the [Slot API](https://github.com/w3c/webcomponents/blob/gh-pages/proposals/Slots-Proposal.md) and the `is` special attribute. However, there are a few key differences:
 
-1. The Web Components Spec has been finalized but is not natively implemented in every browser. Safari 10.1+, Chrome 54+ and Firefox 63+ natively support web components. In comparison, Vue components work consistently in all supported browsers (IE11 with compatibility build and above). When needed, Vue components can also be wrapped inside a native custom element.
+1. The Web Components Spec has been finalized but is not natively implemented in every browser. Safari 10.1+, Chrome 54+ and Firefox 63+ natively support web components. In comparison, Vue components work consistently in all supported browsers (except Internet Explorer 11 - please check details [here](https://github.com/vuejs/rfcs/blob/master/active-rfcs/0038-vue3-ie11-support.md)). When needed, Vue components can also be wrapped inside a native custom element.
```

Comment:
I'm not sure what this means. Neither Web Components nor Vue 3 support IE 11, so why do we need to mention it as an exception?

Reply:
@skirtles-code  We say `all supported browsers`, and AFAIL Microsoft stops the official support for IE11 only on  Aug 17, 2021. Until then, when we speak about supported browsers, we need to mention IE11 as an exception. What do you think?